src/elm/Page/Community.elm

```diff
@@ -587,14 +592,14 @@ viewAction loggedIn metadata maybeDate action =
 
         usagesColor : String
         usagesColor =
-            if action.usagesLeft >= 1 then
+            if action.usagesLeft >= 1 || action.usages == 0 then
```

Comment:
@lucca65  won't this display actions without usages as though they can still be used?

Reply:
I don't know, in my head this goes like this: we have two fields limit action usages. One is the max `usages` and the other is the usages `usagesLeft` that will be equal or smaller than `usages`.

Maybe I missed something because this is an `if` for adding a class color, but the idea is to show the action as claimable even when it don't have usages validation (usages validation disabled is the same as `usages == 0`

Reply:
@lucca65  I'll take a min to walk through this slowly and get back to you

Reply:
Hey @lucca65  your logic works out on this one, dare I say it's even better and covers all the edge cases